### ISSUE TITLE
hammered+lava crystal on gravel turns to concrete instead of glass

### ISSUE THREAD

**Issue by usafphoenix** [SlimeKnights/TinkersConstruct#1800]
tools with auto-smelt and hammered modifiers not "hammering" before auto-smelt takes affect and nullifies effect of hammered-modifier.

So with newest versions of TC, exastris, and exnihilio, 
some of my tools (hammer and excavator) have hammered and auto-smelt on them, 

1) when the hammer is used on stone or cobble, it turns into smooth stone
(cobble gets smelted as opposed to hammered, then smelted. hammering has no effect)

2) when the excavator is used on gravel, it turns to concrete, rather than sand---and then glass. (as i thought it would/ [was designed to].

3) using excavator on sand turns it to glass. no reduction to dust. (nothing to smelt, there, anyways)

basically, i'm guessing that auto-smelt is overridding whatever action/function call the hammered modifier uses. Either modifier appears to function fine as long as they aren't on the same tool, but autosmelt will cause the slot used by "hammered" modifier to be nullified/useless.

**Comment by usafphoenix** [comment]
guess technically the modifier is called "smashing". same difference.

**Comment by bonii-xx** [comment]
Not supported
